client/src/components/ProtectedRoute/ProtectedRoute.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Redirect, Route, RouteProps } from 'react-router';
+import { useAuth } from '../../context/useAuthContext';
+
+export interface IPrivateRouteProps extends RouteProps {
+  redirectPath: string; // redirect path if don't authenticate route
+}
+
+const PrivateRoute: React.FC<IPrivateRouteProps> = (props) => {
+  const { loggedInUser } = useAuth();
+
+  return loggedInUser?.email ? (
+    <Route {...props} component={props.component} render={undefined} />
```

Comment:
Questions: Why is the render attribute 'undefined' here? :)

Reply:
the component prop renders the component

Reply:
In that case you can remove the render prop here

Reply:
yh did that